### PR TITLE
feat(actions): add place bid action 

### PR DESCRIPTION
**Pull request by kurpav**
- added action to place bid
- tested place bid transaction instructions in metaplex-foundation repo

### CODE REVIEW

File: src/actions/placeBid.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      lamports: amount.toNumber() + accountRentExempt * 3,
+      mint: NATIVE_MINT,
+    },
+  );
+  const closeTokenAccountTransaction = new Transaction().add(
+    Token.createCloseAccountInstruction(
+      TOKEN_PROGRAM_ID,
+      payingAccount.publicKey,
+      bidder,
+      bidder,
```
Comment (b2kdaman) [inline src/actions/placeBid.ts:147]: Is this OK?

**Review by b2kdaman — CHANGES_REQUESTED** [review]

**Review by m-sebastiyan — APPROVED** [review]

File: src/actions/placeBid.ts (hunk 1 above)
Comment (kurpav) [reply src/actions/placeBid.ts:147]: Do you mean two params `bidder` ?

Comment (b2kdaman) [reply src/actions/placeBid.ts:147]: Yes

Comment (kurpav) [reply src/actions/placeBid.ts:147]: Yes, the params passed correctly

**Review by b2kdaman — APPROVED** [review]

**Pull request merged**